docs/contributors/opentracingprovider.adoc

```diff
@@ -188,4 +189,34 @@ When the ‘inject’ method is called on the OpenTracing Tracer, it will provid
 An unique id should be created, associated with the trace state being returned for injection into the outbound message, and used to create an `Interaction` based correlation id for the current node. The node should also be created as a `Producer`.
 
 
+=== Sampling 
 
+Sampling implementation is not required, however if a provider decideds to implement it should follow following rules.
```

Comment:
I think that openTracing provider can be used without sampling, and probably users can provide their own sampling algorithms. 

Or maybe should we say that if the sampling is not implemented that provider should explicitly set  `ReportingLevel` to `All`?